### DISCUSSION TITLE
Normalize Icon Size in Taskbar

### DISCUSSION THREAD

**Discussion by bradsmithwv** [lawand-dot-io/taskbar#98] [Feature requests]
I see you have a planned feature to scale the taskbar size, this is appreciated!  However, I have noticed that the mac dock seems to normalize the icon size with one another.  The Taskbar icon sizes do not seem to be normalized.  Please see images below.  Notice (for example) the Edge and Messages icons seem different sizes in Taskbar but seem the same size in dock.

<img width="280" alt="image" src="https://github.com/user-attachments/assets/06ba29cf-1dde-4a55-bbef-6897b789e6cd">

<img width="404" alt="image" src="https://github.com/user-attachments/assets/d91f654c-c7f3-44d1-8c3e-ab3b752ea6b3">

**Comment by lawand-dot-io** [comment]
Hello. Thanks for the request. This is a duplicate of https://github.com/lawand-dot-io/taskbar/issues/47 and I completely agree with you that it should be solved. I will try to find a solution for this. Thanks and enjoy Taskbar :)
